### DISCUSSION TITLE
Rules specific value route parameter added but still succeeded use other value

### DISCUSSION THREAD

**Discussion by consciousnessdev** [mockoon/mockoon#493] [Help] [answered]
I setup Delete method request success with rules route parameter id with value = 10, but when test with id value = 11 it still succeeded (200)

**To Reproduce**
Steps to reproduce the behavior.
Route URI : localhost:3001/users/:id
Method Request : DELETE
Rules Route Parameter : id = 10

**Expected behavior**
When send request method DELETE with id route parameter value 11
URI: localhost:3001/users/11

it must fail to response 404 with response
```json
{
  "code": 404,
  "success": false,
  "message": "Fail to delete user. User not found!"
}
``` 

**Screenshot**
- Request Screenshot
![image](https://user-images.githubusercontent.com/30996656/120417377-efbc5d80-c388-11eb-8f34-c36f2369c600.png)
Note:
Response Code : 200
Rules : Route Params key id with value 10

- Environment Log
![image](https://user-images.githubusercontent.com/30996656/120417713-812bcf80-c389-11eb-90d6-8c87a28cec15.png)

**Mockoon version:**
1.14.1

**OS / OS version:**
Linux Ubuntu 21.04

**Comment by 255kb** [comment, marked as answer]
This is probably due to the fact that your 200 response is the first one, so it's considered as default and served if no rule matches.
Your setup should be inverted, the 404 first, catching all the cases, the 200 second, with the `id=10` rule. 
See https://mockoon.com/docs/latest/route-responses/multiple-responses/#default-route-response for more information. 

This system will probably be changed in the future (#252) as drag and drop is not really obvious.
